openbb_platform/providers/fmp/openbb_fmp/models/financial_ratios.py

```diff
@@ -24,22 +27,30 @@ class FMPFinancialRatiosQueryParams(FinancialRatiosQueryParams):
     Source: https://financialmodelingprep.com/developer/docs/#Company-Financial-Ratios
     """
 
+    __json_schema_extra__ = {"symbol": ["multiple_items_allowed"]}
+
     period: Literal["annual", "quarter", "ttm"] = Field(
         default="annual", description=QUERY_DESCRIPTIONS.get("period", "")
     )
+    with_ttm: Optional[bool] = Field(
```

Comment:
This was purposefully left as an either/or situation.  Combining "TTM" with "quarter" or "annual" makes a huge mess of the response and they don't necessarily play nicely together.

![Screenshot 2024-04-19 at 4 28 58 PM](https://github.com/OpenBB-finance/OpenBBTerminal/assets/85772166/142e73fd-3bf9-469f-8da0-ce04a03f8664)